crates/swc_css_modules/tests/with-compat/vercel/010/output.css

```diff
@@ -0,0 +1,3 @@
+.__local__quickstart .__local__buttons  > * {
```

Comment:
There is extra space and looks like we can cut them more, but it is not critical, I will refactor it so we will fix it

Reply:
Cool, thank you!